kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -445,6 +447,12 @@ public void channelInactive(ChannelHandlerContext ctx) {
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
         LOGGER.warn("Netty caught exception from the frontend: {}", cause.getMessage(), cause);
+        if (cause instanceof DecoderException de && de.getCause() instanceof FrameOversizedException e) {
+            LOGGER.warn(
```

Comment:
If TLS is configured and we reach here, the TLS must be established.   So how about only mentioning the TLS aspect if it is an actual possible root cause? 

(aside: we should probably write a summary of what virtualclusters are listening to what ports on startup and if TLS is configed.  That'd be really useful to have when trying to diagnose a problem).

```
            var tlsHint = virtualCluster.getUpstreamSslContext().isPresent() ? "" : " or an unexpected TLS handshake";
            LOGGER.warn(
                    "Received over-sized frame, max frame size bytes {}, received frame size bytes {} "
                            + "(hint: are we decoding a Kafka frame, or something unexpected like an HTTP request{})?",
                    e.getMaxFrameSizeBytes(), e.getReceivedFrameSizeBytes(), tlsHint);
```

Reply:
<nonblocking>Ideally we'd only  write the hint if the connection is new.  Once we've successfully processed the first frame, it is no longer appropiate</nonblocking>

Reply:
I think that's supposed to be the downstream SSL context, right? We are looking for a TLS conn between client and proxy.